src/app/@ansyn/imagery/model/visualizers/initial-annotations-feature-properties.ts

```diff
@@ -0,0 +1,43 @@
+import { IVisualizerStyle, MarkerSize } from './visualizer-style';
+import { cloneDeep } from 'lodash';
+
+const ANNOTATIONS_INITIAL_STYLE: IVisualizerStyle = {
+	stroke: '#27b2cfe6',
```

Comment:
use ; instead of , for all properties

Reply:
this is not an interface so it should be ,